doc/source/developing/testing.rst

```diff
@@ -387,10 +387,17 @@ Here is an example test function:
    def test_my_ds():
        ds = data_dir_load(my_ds)
 
-       def create_image(filename_prefix):
-           plt.plot([1, 2], [1, 2])
-           plt.savefig(filename_prefix)
-       test = GenericImageTest(ds, create_image, 12)
+       def create_images(filename_prefix):
+            plot = yt.SlicePlot(ds, "z", "density")
+            plot.save("%s_log" % filename_prefix)
+
+            plot.set_log("density", False)
+            plot.save("%s_lin" % filename_prefix)
+
```

Comment:
I don't think that this change really enhances this part of the documentation. are you trying to show that you can have multiple yt plot objects or something?

Reply:
Yes that was the idea, because I only grasped it from reading the source code. But since you think the note is enough I will revert this.

Reply:
I personally think the note is helpful. Thank you for reverting it! I think we should have a community discussion about doc improvements after the 3.6 release; you've had a lot of good feedback about the docs and I think it warrants a bigger discussion!